renderdoc/core/target_control.cpp

```diff
@@ -778,6 +808,16 @@ struct TargetControl : public ITargetControl
       reader.EndChunk();
       return msg;
     }
+    else if(type == ePacket_CapturableWindowCount)
+    {
+      msg.type = TargetControlMessageType::CapturableWindowCount;
+      int windows = 0;
```

Comment:
When serialising always use an explicitly sized type to avoid any chance that the type might be defined differently on different platforms. Here that would be `uint32_t`.